app/controllers/application_controller.rb

```diff
@@ -82,7 +86,10 @@ def authenticate_user!
 
   def authenticate_admin_user!
     authenticate_user!
-    unless current_user.admin?
+
+    if not_signed_in?
+      render nothing: true, status: 403
```

Comment:
right you are! have a test for it but circle is down so I didn't realize it was failing. good :baseball: